common/components/common/integrations/PaypalDonationButton.jsx

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import React from 'react';
+import cdn,{Images} from "../../utils/cdn.js";
+import url from "../../utils/url.js";
+
+export const OtherAmountSelected: string = "OTHER";
+
+type Props = {|
+  donateAmount: ?string,
+  donateMonthly: ?boolean
+|};
+
+class PaypalDonationButton extends React.Component<Props> {
+  constructor(): void {
+    super();
+  }
+  
+  render(): React$Node {
+    return (
+      <div className="PaypalDonationButton">
+        <form action={window.PAYPAL_ENDPOINT} method="post" target="_top">
+          {this._renderFormHiddenFields()}
+        
+          <input type="image" src={cdn.image(Images.PAYPAL_BUTTON)} border="0"
+                 name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate with PayPal button"/>
+          <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1"/>
+        </form>
+      </div>
+    );
+  }
+    
+  _renderFormHiddenFields(): $ReadOnlyArray<React$Node> {
+    let fields:Array<React$Node> = [
+      <input type="hidden" name="cancel_return" value="/index/" />,
+      <input type="hidden" name="return" value={url.hostname() + "/index/?section=ThankYou"} />,
```

Comment:
Here's the steps to reproduce:

1. go to donate page on dev
2. pick a pre-selected amount or Other, doesn't matter, click orange donate button
3. on  Paypal's site click Donate with Paypal
4. log in with sandbox account
5. click Donate Now to confirm donation
6. on the page which says "You've donated $x.xx USD to Mark Frischmuth's Test Store" don't click the return link. Instead, wait for it to redirect you like it says it will "in a few seconds" - takes less than 10.  You'll get an infinite spinner and the address bar will read https://www.sandbox.paypal.com/donate/member/democracy-lab-dev.herokuapp.com/index/?section=ThankYou

Gotten this in both Firefox and Chrome on two different computers.